modular_chomp/code/modules/mob/living/simple_mob/butchering.dm

```diff
@@ -0,0 +1,29 @@
+//I don't know how to add more material currently so uhhh, that will be for later.
+/mob/living/simple_mob/vore/bigdragon
+	meat_type = /obj/item/weapon/reagent_containers/food/snacks/dragonmeat
+
+	//butchery_loot = list(\

```

Comment:
not sure what the backslashes here are for.
Block comment with /* */ if that's your intention

Reply:
Pretty much was trying to set up the groundworks for a future PR, but seeing as what I attempted is causing issue removed that it. The //butchery_loot.